src/views/Home/Home.jsx

```diff
@@ -0,0 +1,178 @@
+import Hero from '../../components/Hero';
+import Modal from '../../components/Modal';
+
+axios.interceptors.request.use((config) => {
+  const newConfig = {
+    ...config,
+    headers: {
+      ...config.headers,
+      env: process.env.REACT_APP_ENV || 'development',
+    },
+  };
+
+  return newConfig;
+});
+
+function Home() {
+  const [state, setState] = useState({
+    clientId: process.env.REACT_APP_OAUTH_GOOGLE_ID,
+    attractions: [],
+    destination: '',
+    isLoading: false,
+    modal: {
+      show: false,
+      attraction: {},
+    },
+    noResults: false,
+  });
+
+  const [features, setFeatures] = useState({
+    loading: true,
+    flags: {},
+  });
+
+  const Feature = feature(features.flags, features.loading);
+
+  // For loading feature flags
+  useEffect(() => {
+    const getFlags = async () => {
+      const promise = new Promise((resolve) => {
+        setTimeout(() => {
+          resolve({
+            'profile-link': false,
+            profile: true,
+          });
+        }, 500);
+      });
+
+      const flags = await promise;
+
+      setFeatures({
+        flags,
+        loading: false,
+      });
+    };
+
+    getFlags();
+  }, []);
+
+  useEffect(() => {
+    const token = store.get();
+    if (token) {
+      setState(prevState => ({
+        ...prevState,
+        loggedIn: true,
+      }));
+    }
+
+    // TODO: This is temporary tracking to validate setup
+    track.pageview('/');
+    track.event({
+      category: 'Main',
+      action: 'Generic Action',
+    });
+  }, []);
+
+  // performs action when user clicks the ROAM button
+  const handleSearch = (destination) => {
+    setState(prevState => ({
+      ...prevState,
+      isLoading: true,
+      destination,
+    }));
+    // request to backend that will request to API and send back the data
+    axios.get(`${process.env.REACT_APP_ENDPOINT}/places/details/${destination}`)
+      .then(({ data: { places } }) => {
+        if (places) {
+          setState(prevState => ({
+            ...prevState,
+            attractions: places,
+            isLoading: false,
+            noResults: false,
+          }));
+        } else {
+          setState(prevState => ({
+            ...prevState,
+            attractions: [],
+            isLoading: false,
+            noResults: true,
+          }));
+        }
+      })
+      .catch((error) => {
+        console.log(error, 'here');  // eslint-disable-line
+      });
+  };
+
+  const showModal = async (place) => {
+    setState(prevState => ({
+      ...prevState,
+      modal: {
+        ...prevState.modal,
+        show: true,
+        attraction: place,
+      },
+    }));
+  };
+
+  const closeModal = () => {
+    setState(prevState => ({
+      ...prevState,
+      modal: {
+        ...prevState.modal,
+        show: false,
+        attraction: {},
+      },
+    }));
+  };
+
+  const classTest = !state.modal.show ? styles.App : `${styles.App} ${styles.blur}`;
```

Comment:
I am 95% certain I am responsible for this horrible name and forgot to update it, but we can change this from classTest to something more meaningful like wrapper.

We will also likely have to move this and the assignment on line 42 up to App.jsx to handle blurring the background both here and in profile when clicking on favorites.